### ISSUE TITLE
Rename main branch

### ISSUE THREAD

**Issue by dthaler** [openenclave/openenclave#4393]
To follow latest inclusive language best practices, we should rename the "master" branch to "main"

More info:
* https://github.com/github/renaming
* https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/

**Comment by radhikaj** [comment]
Agree with this, will break CI etc.

**Comment by radhikaj** [comment]
We can start by creating a main branch.

**Comment by dthaler** [comment]
> We can start by creating a main branch.

Per the github instructions, it's easier to rename the existing branch rather than creating a new one.   Github actions CI/CD should not break.   We've done this for other repos.

**Comment by CyanDevs** [comment]
I agree we should rename the branch to main. This will require some coordinated changes in Jenkins CI as well so please make sure to loop @rs-- and I in.
